stable/spark-history-server/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: spark-history-server
-version: 0.4.1
+version: 0.4.2
```

Comment:
Let's make the version 0.5.0 because this is not a bug fix, but adding a new feature.

Reply:
Done